docs/source/install/Developers.rst

```diff
@@ -0,0 +1,125 @@
+Installation for Developers
+===========================
+
+In order to start developing, clone the QMAP repository using
+
+    .. code-block:: console
+
+        $ git clone --recurse-submodules -j8 https://github.com/cda-tum/qmap
+
+Note the :code:`--recurse-submodules` flag. It is required to also clone all the required submodules. If you happen to forget passing the flag on your initial clone, you can initialize all the submodules by executing :code:`git submodule update --init --recursive` in the main project directory.
+
+A C++ compiler supporting *C++17* and a minimum CMake version of *3.14* is required to build the project.
```

Comment:
This is missing the additional requirement on the Z3 library and boost program options. The respective text can be copied/adapted straight from the README file.

Reply:
Now it is indeed included.